html/user/buda_submit.php

```diff
@@ -252,6 +252,9 @@ function create_jobs(
     if ($keywords) {
         $cmd .= " --keywords '$keywords'";
     }
+    if ($user->seti_id) {
```

Comment:
is `seti_id` a correct value?
you check for it but later pass just `id`

Reply:
We use user.seti_id as a boolean flag for 'use only my computers'.